src/ExpandableDataPicker/ExpandableDataPicker.js

```diff
@@ -70,6 +70,7 @@ module.exports = kind(
 	*/
 	collectionChanged: function () {
 		this.$.list.set('collection', this.collection);
+		if (this.collection && !this.$.list.hasNode()) this.$.list.refresh();
```

Comment:
I think the benefit of ExpandableDataRepeater is that it leverages a Collection to generate the children rather than requiring a static list (or manually managed list). I'll add that this change *does not* force a render but it will create the controls. There is still a cost, of course, but it's minimized. The rendering delay is managed by the drawer set to `renderOnShow: true`.